assets/js/components/DetailsPermaLinks.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * Details Permalink Links component.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import { Fragment } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import { CORE_SITE } from '../googlesitekit/datastore/site/constants';
+import Link from './Link';
+const { useSelect } = Data;
+
+export default function DetailsPermaLinks( { title, path } ) {
+	const siteURL = useSelect( ( select ) => select( CORE_SITE ).getReferenceSiteURL() );
+	const permaLink = new URL( path, siteURL ).href;
+	const detailsURL = useSelect( ( select ) => {
+		return select( CORE_SITE ).getAdminURL( 'googlesitekit-dashboard', { permaLink } );
+	} );
+
+	return (
+		<Fragment>
+			<Link
+				className="googlesitekit-display-block"
+				href={ detailsURL }
+				inherit
+			>
+				{ title }
+			</Link>
+
+			<Link
```

Comment:
There appears to be a problem here; these two links can currently appear in the same row (e.g. in the WP dashboard widget) which would be a regression and looks broken.

Reply:
This was due to missing styles since wpdashboard has a separate stylesheet which I've updated.

Also, related to this was that I noticed the widget app's render target did not have the `.googlesitekit-plugin` class which was preventing some styles from being applied as well, so I added this too. This resulted in a slight difference in the font rendering in the widget (since core uses a different font by default) but it seems more likely that this was missed rather than an intentional omission (let me know if that's not a correct assumption).